examples/data-objects/multiview/slider-coordinate-view/package.json

```diff
@@ -11,14 +11,22 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
```

Comment:
Same comment as in other PRs. If this pattern is in use ignore my comments and we can clean up